services/app-api/forms/naaar.json

```diff
@@ -256,12 +256,16 @@
                     {
                       "id": "uITThePQiXntwGGViPTD62",
                       "label": "Specialist",
-                      "hint": "Include all specialists (except for Mental health) within this category. You’ll be able to specify them when creating standards."
+                      "hint": "Include all specialists (except for Mental health) within this category."
```

Comment:
these hints don't show up in the report for me. testing in the deployed env and created two new reports

| FIGMA | Deployed env |
| ------- | ------------- |
| ![Screenshot 2025-06-11 at 11 10 19 AM](https://github.com/user-attachments/assets/f6777a63-16c6-4405-866a-ca4e2fbb8485) |  ![Screenshot 2025-06-11 at 11 10 16 AM](https://github.com/user-attachments/assets/50859fa3-ed38-4b4b-9a74-b9af40bb7a4e) |